common/src/main/java/com/wynntils/services/lootrunpaths/LootrunPathFileParser.java

```diff
@@ -121,10 +124,12 @@ public static LootrunSaveResult writeJson(UncompiledLootrunPath activeLootrun, F
             }
             json.add("notes", notes);
 
-            json.addProperty(
-                    "date",
-                    DateFormat.getDateTimeInstance(DateFormat.DEFAULT, DateFormat.DEFAULT, Locale.US)
-                            .format(new Date()));
+            ZonedDateTime time = ZonedDateTime.ofInstant(Instant.now(), ZoneId.systemDefault());
+
+            DateTimeFormatter formatter =
+                    DateTimeFormatter.ofLocalizedDateTime(FormatStyle.MEDIUM).withLocale(Locale.US);
+
+            json.addProperty("date", time.format(formatter));
```

Comment:
Can this load the old format properly? I think this is the place @magicus mentioned, where we needed to have a kind of backward-compatibility.

Reply:
I think I did check that the format was the same but I'll give it a double check.

Do we support legacy lootruns I can't remember, they might be different

Reply:
I think we can load them, if they don't have notes.

Reply:
The paths I have on my main instance load fine so they should be compatible

Reply:
In fact, I think this is the only place we actually use the date and it's just for saving, it's never read anywhere